charts/openstack-actions-runner/templates/service-manager.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "openstack-actions-runner.fullname" . }}-manager
+  labels:
+    role: manager
+    {{- include "openstack-actions-runner.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.deployment.service.type }}
+  ports:
+    - name: metrics
+      port: {{ .Values.metricsPort }}
+      protocol: TCP
+  selector:
+    role: manager
+    {{- include "openstack-actions-runner.selectorLabels" . | nindent 4 }}
```

Comment:
Creating a service so that the service monitor object can discover the endpoint to monitor